src/codeu/chat/client/simplegui/ChatGuiFX.java

```diff
@@ -131,37 +119,41 @@ public void start(Stage primaryStage) throws Exception {
 
         // Main Page
 
-        hboxClient = new HBox();
-        hboxInput = new HBox();
-        userVBox = new VBox();
-        chatVBox = new VBox();
-        convosVBox = new VBox();
-        container = new BorderPane();
-        sendButton = new Button("Send");
-        updateButton = new Button("Update");
-        addConvoButton = new Button("Add Conversation");
-        userTitle = new Text("Users");
-        chatTitle = new Text("Conversation"); // changed based on name?
-        convosTitle = new Text("Conversations");
-        userTf = new TextFlow(userTitle);
-        chatTf = new TextFlow(chatTitle);
-        convosTf = new TextFlow(convosTitle);
-        input = new TextField();
+        HBox hboxClient = new HBox();                           // holds the client
```

Comment:
I would at least try to split the different screens into different functions. Something like:

```java
public void run(String[] args){
   buildLoginScene();
   buildMainScene();
}

private void buildLoginScene(){
   //build login scene
}

private void buildMainScene(){
   //build main scene
}
```